sql/src/test/java/io/crate/protocols/postgres/ClientMessages.java

```diff
@@ -147,4 +147,9 @@ static void sendPasswordMessage(ByteBuf buffer, String password) {
         buffer.writeBytes(passwordBytes);
         buffer.writeByte(0);
     }
+
+    static void sendTermination(ByteBuf buffer) {
```

Comment:
shall we comment the purpose of this (testing etc)?

Reply:
@andreidan Actually the whole class is only for testing purposes so....